modules/memberships/class-jetpack-memberships.php

```diff
@@ -250,16 +250,19 @@ public function render_button( $attrs ) {
 		}
 		$button_styles = implode( ';', $button_styles );
 		add_thickbox();
+		global $wp;
 		return sprintf(
-			'<div class="wp-block-button %1$s"><a role="button" data-blog-id="%2$d" data-powered-text="%3$s" data-plan-id="%4$d" data-lang="%5$s" class="%6$s" style="%7$s">%8$s</a></div>',
+			'<div class="wp-block-button %1$s"><a role="button" %10$s href="https://subscribe.wordpress.com/memberships/?blog=%2$d&plan=%4$d&lang=%5$s&pid=%9$d&redirect=%3$s" class="%6$s" style="%7$s">%8$s</a></div>',
 			esc_attr( Jetpack_Gutenberg::block_classes( self::$button_block_name, $attrs ) ),
 			esc_attr( $data['blog_id'] ),
-			esc_attr( $data['powered_text'] ),
+			esc_attr( rawurlencode( home_url( $wp->request ) ) ), // Needed for redirect back in case of redirect-based flow.
```

Comment:
Would `get_permalink()` be a potential replacement here, or am I missing a possible use-case where it's not enough?

Reply:
We haven't shipped the Widget (https://github.com/Automattic/jetpack/pull/12684) yet, but this is something we would like to do at some point.
Aside from that, I do not know if there is a way to launch the editor outside of the post context. Is there?

Reply:
Oh cool, I forgot that was on the roadmap! 👍